src/app/shared/models/bet.model.ts

```diff
@@ -0,0 +1,10 @@
+import { BetStatus } from './bet-status.model';
+
+export interface Bet {
+  favorite_id: number;
+  match_id: number;
+  user_id: number;
+  amountMoney: number;
```

Comment:
Keep code consistent. use `lowerCamelCase` or `snake_case` but not mix them across your project

Reply:
done